doc/Overview/Mvux/ListFeeds.md

```diff
@@ -0,0 +1,77 @@
+---
+uid: Overview.Mvux.ListFeeds
+---
+
+# What are List-Feeds?
+
+The `IListFeed<T>` is like a [Feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes, but is specialized for handling collections.
+
+In Feeds, each data request from the service, returns one single item, whereas in List-Feed each request returns a collection of items.  
+Its distinctive feature is the ability to use its [Operators](xref:Overview.Mvux.ListFeeds#operators) directly on its items, rather than on the entire collection.
+
+Another important thing to remember is that when the service returns an empty collection, it's treated as an Empty message, and the data axis Option will be `None`, although the result was not `null`.
```

Comment:
```suggestion
Another notable characteristic of List-Feed is how it handles the absence of data. When an empty collection is returned by the service, it's treated as an Empty message. The returned data axis Option will be `None`, even though the result was not `null`.
```

Nit: This sentence explains the empty collection scenario well, but seems crowded and may be difficult to follow. I broke it down into smaller sentences.

Reply:
We should also touch on the rationale for the use of `None` when both null or empty. Even if we just link to an anchor on another page, it would help to have an explanation for why this is by design.

Reply:
Addressed